src/components/Progress/ProgressRing.js

```diff
@@ -20,7 +20,8 @@ const getPath = (progress, isComplete) => {
   if (!isComplete) {
     const deg = 2 * Math.PI * progress;
     const isLong = progress >= 0.5 ? '1' : '0';
-    d = `${prefix} ${isLong} 1 ${Math.cos(deg)} ${Math.sin(deg)} ${postfix}`;
+    if (variant === 'fill') d = `${prefix} ${isLong} 0 ${Math.cos(deg)} ${-Math.sin(deg)} ${postfix}`;
```

Comment:
You could utilise the ternary operator here.

```
if (...) ...;
else ...;
```

is a notation you do not commonly see in Javascript